src/tests/nativeaot/SmokeTests/Reflection/Reflection.cs

```diff
@@ -63,6 +63,7 @@ private static int Main()
         TestInvokeMemberParamsCornerCase.Run();
         TestDefaultInterfaceInvoke.Run();
         TestCovariantReturnInvoke.Run();
+        TestGetMemberWithSameMetadataDefinitionAs.Run();
```

Comment:
I do not think we need the regression test for this corner case here. It is ok to depend on the libraries tests for this coverage.